control-plane/catalog/to-consul/syncer.go

```diff
@@ -17,6 +19,33 @@ import (
 	"github.com/hashicorp/go-hclog"
 )
 
+var (
+	baseName            = []string{"consul", "sync_catalog", "to_consul"}
+	registerName        = append(baseName, "register")
+	deregisterName      = append(baseName, "deregister")
+	registerErrorName   = append(baseName, "register", "error")
+	deregisterErrorName = append(baseName, "deregister", "error")
+)
+
+var SyncToConsulCounters = []prometheus.CounterDefinition{
+	{
+		Name: registerName,
+		Help: "Increments for each service registered to Consul via catalog sync",
```

Comment:
Are these metrics meant to count "service" or "service instance"? "Service" implies to me a higher level construct that can consist of multiple instances.

Reply:
Agreed, it sounds like this should be new service instance.